amp/pom.xml

```diff
@@ -1600,6 +1600,27 @@
                 </executions>
             </plugin>
 
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>1.1.1</version>
+                <executions>
+                    <execution>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>java</goal>
+                        </goals>
+                        <configuration>
+                            <mainClass>org.dgfoundation.amp.asciidoc.ApiErrorAsciiDocCreator</mainClass>
+                            <arguments>
+                                <argument>${project.basedir}/src/main/asciidoc</argument>
```

Comment:
Temporary apierrors.doc is generated and written in source folders. For such purposes we should use target folder (or better ${project.build.directory}). After compile phase I can see this file as unstaged in git.